.envrc

```diff
@@ -1,8 +1,7 @@
 ### General env vars ###
 export WORKSPACE_ROOT=$(pwd)
 export RUST_LOG_FORMAT=devel
-export RUST_BACKTRACE=1
```

Comment:
I find the backtraces more obfuscating than informative in most test scenarios, they should be opt-in instead of opt-out.